### DISCUSSION TITLE
How to Update?

### DISCUSSION THREAD

**Discussion by pkscout** [ESPresense/ESPresense#1089] [Q&A]
I feel like a complete idiot, but I can't figure out how to update my devices short of reflashing them and setting them up from scratch again.  I turned on auto update and rebooted one of the devices.  Nothing.  There is an option to put a URL in an use that for update at next restart, but the docs don't mention that option, so I don't know what to put there.

**Comment by DTTerastar** [comment]
You can put PRESS in there and reboot and it'll attempt an auto update.  Or press the update button in Home Assistant.
